spec/models/user_spec.rb

```diff
@@ -111,7 +111,7 @@
   it "shows a user is recent or not" do
     user = create(:user, :created_at => Time.current)
     expect(user.is_new?).to be true
-    user = create(:user, :created_at => Time.current - (User::NEW_USER_DAYS.days + 1))
+    user = create(:user, :created_at => Time.current - (User::NEW_USER_DAYS.days + 1.day))
```

Comment:
Sorry about this bug - it was fixed in another commit, so if you want to merge master into your PR you won't need to make this change.

Reply:
Yep, sure.